Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelemetryOpsInterface.java

```diff
@@ -0,0 +1,8 @@
+package tddmicroexercises.telemetrysystem;
+
+public interface TelemetryOpsInterface {
+
+    void checkTransmission() throws Exception;
+
+    Object getDiagnosticInfo();
```

Comment:
TelemetryOpsInterface dealing with diagnosticInfo
where as TelemetryDiagnosticInterface is dealing with connect and disconnect 
Rename the interface names according to there functionality